include/revng/BasicAnalyses/AdvancedValueInfo.h

```diff
@@ -207,6 +211,10 @@ class Analysis
   std::map<llvm::Instruction *, ConstantRangeSet> InstructionRanges;
   std::set<CFGEdge> TargetEdges;
   std::set<llvm::BasicBlock *> WhiteList;
+  mutable llvm::DenseMap<
+    llvm::BasicBlock *,
+    std::unique_ptr<llvm::SmallVector<llvm::BasicBlock *, 2>>>
```

Comment:
Use `std::vector`?

Reply:
forcing a dense map to have items with the size of a pointers is a 20 second time save when lifting bash (out of 3 minutes and 20 seconds without this)

Reply:
Gosh.
OK, leave a through comment about this.